src/node_modules/app/tsservices/index.js

```diff
@@ -0,0 +1,49 @@
+function ts () {
+	return window.ts;
+}
+
+const LIBDTS = `// Standard types
+interface Date {}
+
+type Record<K extends keyof any, T> = {
+    [P in K]: T;
+};`;
+
+const servicesHost = files => ({
+	getScriptFileNames: () => Object.keys(files),
+	getScriptVersion: () => "0",
+	getScriptSnapshot: fileName => {
+		if (fileName in files) {
+			return ts().ScriptSnapshot.fromString(files[fileName]);
+		}
+		if (fileName === 'lib.d.ts') {
+			return ts().ScriptSnapshot.fromString(LIBDTS);
+		}
+		return undefined;
+	},
```

Comment:
here I'd prefer using something like this:

```suggestion
	getScriptSnapshot: fileName => {
		let scriptSnapshot;
		
		if (fileName in files) {
			scriptSnapshot = ts().ScriptSnapshot.fromString(files[fileName]);
		} else if (fileName === 'lib.d.ts') {
			scriptSnapshot = ts().ScriptSnapshot.fromString(LIBDTS);
		}
		
		return scriptSnapshot;
	},
```